libraries/botbuilder-core/package.json

```diff
@@ -21,6 +21,7 @@
   "typings": "./lib/index.d.ts",
   "dependencies": {
     "assert": "^1.4.1",
+    "botframework-connector": "4.1.6",
```

Comment:
We've avoided adding `botframework-connector` to `botbuilder-core` because of its various dependencies that run only on the Node.js runtime. `botbuilder-core`, `botbuilder-ai`, `botbuilder-dialogs` and other packages have been shipped with a "runnable-in-Node.js-or-browsers" stance; which means we can't add `botframework-connector` as a dependency.

Ideally the package/code in `botframework-connector` would have been factored or shipped (bundled/packed) in a way to allow for the browser-compatible packages to take a direct dependency on the necessary code.

There's a few paths we could take to safely consume the necessary `SkillValidation` code in `botbuilder-core`:

1. Factor out the code into a new package that's consumable by any environment
2. Duplicate the code
3. Add environment integration tests to make sure that botframework-connector is able to be bundled by modern JS toolchains
   3.a. Optionally, we can start including bundled distributions for those wishing to build browser/non-Node environment bots.

Reply:
Okay - I've rebased from `main` which now includes the `browserify`d `botframework-connector` package, so this should be resolved.

Reply:
Perfect, thanks!